pkg/helm/export.go

```diff
@@ -59,6 +64,27 @@ func ExportHelmChart(rootFs, fs billy.Filesystem, helmChartPath string, packageV
 	}
 	chartVersion := chartVersionSemver.String()
 
+	if releaseOptions != nil && releaseOptions.Contains(chart.Metadata.Name, chartVersion) {
```

Comment:
I tried creating a new version of a chart and was trying to see if the icon URL is changed. 

It is not changing since the new version is not yet in release.yaml. Informing all feature chart owners about adding the version beforehand to release.yaml is too complicated I feel. 

If you don't have any other solution, I am okay with adding a simple make target in CI which will check if the chart.Icon is not set to URL or not and if it is set to http(s) URL, the CI will fail.

Reply:
You're right. I changed the code to just use a simple bool flag depending on which command is being ran